packages/embeds/embed-core/src/embed-iframe.ts

```diff
@@ -440,6 +458,13 @@ if (isBrowser) {
   log("Embed SDK loaded", { isEmbed: window?.isEmbed?.() || false });
   const url = new URL(document.URL);
   embedStore.theme = window?.getEmbedTheme?.();
+
+  embedStore.uiConfig = {
+    // TODO: Add theme as well here
+    // theme:
+    layout: url.searchParams.get("layout") as BookerLayouts,
```

Comment:
layout can be configured using query param(which is basically `config.layout` in inline instruction) or UI instruction.